### DISCUSSION TITLE
Anonymize Event

### DISCUSSION THREAD

**Discussion by Kunsi** [pretalx/pretalx#1262] [Ideas] [answered]
It would be nice if events could be anonymized after it has happened. Ideally, pretalx would provide a ui to select which data you want to anonymize. Not all people would want to anonymize all of the data.

Anonymizing would be a one-way, one-time operation, though.

Possible data to anonymize could be:
* Names
* E-Mail addresses
* Answers to (some) questions

Maybe also add an option to only anonymize talks in certain states (for example, anonymize non-accepted talks only).

**Comment by rixx** [comment, marked as answer]
That does sound like a cool feature, and I'd welcome contributions that would add an "archival" action to pretalx! (This would also work as a plugin, but it would probably be better to have this action integrated in pretalx itself, and to allow plugins to extend it.)
